### PR TITLE
Update nginx.conf

### PR DESCRIPTION
**Pull request by ericgaspar**
#484

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: conf/nginx.conf [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Expires mappings
+map $sent_http_content_type $expires {
```
Comment (nicofrand) [inline conf/nginx.conf:2]: From my tests (I hardcoded it on my server) this need to be on the same level as `server {`. Did you test it?

**Comment by ericgaspar** [comment]
!testme

File: conf/nginx.conf (hunk 1 above)
Comment (ericgaspar) [reply conf/nginx.conf]: > Did you test it?
No, I havn't.

Comment (nicofrand) [reply conf/nginx.conf]: I double-checked: http://nginx.org/en/docs/http/ngx_http_map_module.html#map, `map` can only be defined in the `http` context, thus at the same level as the `server`.

**Pull request closed without merging**